src/components/SwapForm/SwapActionButton/SwapOnlyButton.tsx

```diff
@@ -107,6 +107,14 @@ const SwapOnlyButton: React.FC<Props> = ({
       gasUsd: routeSummary?.gasUsd,
       inputAmount: routeSummary?.parsedAmountIn,
       priceImpact: routeSummary?.priceImpact,
+      feeInfo: routeSummary?.fee
+        ? {
+            chargeTokenIn: routeSummary.extraFee.chargeFeeBy === 'currency_in',
```

Comment:
duplicate vs line 112 => write a utils function?

Reply:
Nice. I fixed it